tests/Unit/Settings/SettingsManagerTest.php

```diff
@@ -64,21 +64,19 @@
 });
 
 it('retrieves default settings', function (string $platform) {
-    $resetPlatform = mockPlatform($platform);
+    MockPdkFactory::create(['platform' => $platform]);
```

Comment:
aan het resultaat van de bijgewerkte snapshot zag ik dat dit niet goed ging, dus vandaar dat er grotere snapshot wijzigingen zijn, nu leggen we wél vast wat de platform defaults zijn :)